chess-centre-app/src/components/Navigation/LandingNav.js

```diff
@@ -27,6 +29,26 @@ const LandingNav = (props) => {
     history.push("/");
   };
 
+  useEffect(() => {
+      const getCurrentUser = async () => {
+        try {
+          await Auth.currentAuthenticatedUser();
```

Comment:
This is probably redunant but steps towards removing the manual localStorage session state.